packages/core/src/sharedWishlists/redux/__tests__/reducer.test.js

```diff
@@ -0,0 +1,369 @@
+import * as actionTypes from '../actionTypes';
+import * as fromReducer from '../reducer';
+import reducer, { entitiesMapper } from '../';
+
+let initialState;
+
+describe('shared wishlists reducer', () => {
+  beforeEach(() => {
+    initialState = reducer();
+  });
+
+  describe('reset handling', () => {
+    it('should return the initial state when there are no fields to keep', () => {
+      expect(
+        reducer(undefined, {
+          payload: {},
+          type: actionTypes.RESET_SHARED_WISHLIST_STATE,
+        }),
+      ).toEqual(initialState);
+    });
+
+    it('should only reset to the initial state the fields specified to reset - reducer root (error)', () => {
+      const state = {
+        error: 'Unexpected Error',
```

Comment:
Use an Error instance here please to make this closer to what its value would be.

Reply:
Updated! ✅